fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/rollcall/RollCallViewModel.java

```diff
@@ -51,6 +52,8 @@ public class RollCallViewModel extends AndroidViewModel implements QRCodeScannin
   private final Wallet wallet;
   private final Gson gson;
 
+  private PublicKey key = null;
```

Comment:
I think this would be greatly simplified if the fragment just filtered itself if it wants to exclude the organizer. Moreover having these kind of "CurrentKey" variables updated when some viewmodel action are performed is something we used to have everywhere in view models. It is challenging to maintain as it makes it hard to understand when and why the value was updated. As the complexity grow it is also very easy to make mistakes and have those "CurrentValues" decorrelated with what is shown to the user.

Finally, I think this goes against the general architecture of the app as the displaying update is based on the user pressing "open" rather than the underlying roll-call's state being updated following the reception of the corresponding broadcast message from the backend